app/main.py

```diff
@@ -1,3 +1,15 @@
 def cache(func):
-    # Write your code here
-    pass
+    results = {}
+
+    def inner(*args):
+        results
```

Comment:
I believe this line is extra

Reply:
If I remove this line, flake8 give an error
![image](https://user-images.githubusercontent.com/107579138/177132621-ae3acb73-6cba-444e-af89-1be87d248e33.png)

Reply:
I meant this line:
```
        results
```